packages-next/fields/src/types/decimal/index.ts

```diff
@@ -0,0 +1,29 @@
+import { Decimal } from '@keystonejs/fields';
+import type { FieldType, BaseGeneratedListTypes, FieldDefaultValue } from '@keystone-next/types';
+import { resolveView } from '../../resolve-view';
+import type { FieldConfig } from '../../interfaces';
+
+export type Int = number & { __int__: void };
+
```

Comment:
```suggestion
```

I'd prefer not to use branded types here. While they send a soft signal to the reader, they make for more verbose code for the user, who has to cast their argument to `Int` wherever they use this. They also don't provide any type safety, as the user can case any value to `Int`, whether it's actually an integer value or not.